extensions/wikia/CreateNewWiki/FinishCreateWikiController.class.php

```diff
@@ -105,4 +111,19 @@ public function FinishCreate() {
 		$wgOut->redirect($mainPage.'?wiki-welcome=1');
 	}
 
+	/**
+	 *
+	 * @param $mianPageTitle string
+	 * @param $mainPageArticle Article
+	 */
+	private function initHeroModule( $mianPageTitle, $mainPageArticle ) {
```

Comment:
Frist params name is misspelled, what about adding type hinting for article param